conmon-rs/server/src/config.rs

```diff
@@ -124,9 +127,19 @@ impl Default for Config {
 const SOCKET: &str = "conmon.sock";
 const PIDFILE: &str = "pidfile";
 
+const DEFAULT_RUNTIME: &str = "runc";
```

Comment:
I'm unsure about this. as we move to support podman, their default is crun. we don't have a config value to draw this from. I think we may be better not providing a default

Reply:
Okay sounds good to me.